src/hotspot/share/runtime/globals.hpp

```diff
@@ -1871,7 +1871,7 @@ const int ObjectAlignmentInBytes = 8;
   product(size_t, AsyncLogBufferSize, 2*M,                                  \
           "Memory budget (in bytes) for the buffer of Asynchronous "        \
           "Logging (-Xlog:async).")                                         \
-          range(100*K, 50*M)                                                \
+          range(DEBUG_ONLY(96) NOT_DEBUG(100*K), 50*M)                      \
```

Comment:
Why 96 when the test uses 192? I guess it could be anything.

Reply:
Short answer: It should be 192, I was confused.

Long answer: I picked 96 because that's what I used when hard-coding the buffer size, but I forgot that we have 2 buffers who share `AsyncLogBufferSize` and 192/2 = 96.